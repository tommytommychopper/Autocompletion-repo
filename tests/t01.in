insert bannana 3
insert very 3
insert serious 3
insert verizon 3
insert bank 3
insert address 3
insert verb 3
insert serial 3
insert anchor 3
insert ancient 3
insert bankruptcy 3
insert ancestry 3
insert addiction 3
print_inorder
heightPrint
printBF
findPrint	ser
findPrint	anc
findPrint	ver
findPrint	ban
findPrint	add
