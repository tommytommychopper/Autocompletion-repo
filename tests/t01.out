(add){address addiction } (anc){anchor ancient ancestry } (ban){bannana bank bankruptcy } (ser){serious serial } (ver){very verizon verb } 

0 1 0 2 0 

0 0 0 1 0 

serious serial 
anchor ancient ancestry 
very verizon verb 
bannana bank bankruptcy 
address addiction 
